src/mlpack/methods/preprocess/image_converter_main.cpp

```diff
@@ -2,7 +2,7 @@
  * @file image_converter_main.cpp
  * @author Jeffin Sam
  *
- * A CLI executable to load and save a image dataset.
+ * A CMD executable to load and save a image dataset.
```

Comment:
```suggestion
 * A binding to load and save a image dataset.
```
I think the comment was a little inaccurate to begin with, so maybe we can make it better while we're at it. :)